src/environments/environment.ci.ts

```diff
@@ -50,6 +50,7 @@ export const environment = {
   visualAcuityTest: `Visual Acuity Test`,
   haemoglobinTest: `Haemoglobin Test`,
   parentAPI: `${MMU_API}`,
+  abhaExtension: `@abdm`,
```

Comment:
This can be a variable. 
In non production environments the value will be for sandbox - @sbx